### PR TITLE
update(sonarr): use tar ball for installation to prevent unsupported repo issues

### PR DESCRIPTION
**Pull request by liaralabs**
## Description
Moves sonarr to utilizing the tar ball installation as bullseye does not have a repo to install from, yet it shouldn't matter

## Fixes issues: 
- Bullseye cannot install Sonarr

## Change Categories

- Bug fix 

## Checklist

- [x] Branch was made off the `develop` branch and the PR is targetting the `develop` branch
- [ ] Docs have been made OR are not necessary
    - PR link: 
- [x] Changes to panel have been made OR are not necessary
    - PR link: 
- [x] Code is formatted [(See more)](https://github.com/swizzin/swizzin/blob/master/CONTRIBUTING.md#editor-plugins-and-tooling)
- [x] Code conforms to project structure [(See more)](https://swizzin.ltd/dev/structure)
- [x] Shellcheck isn't screaming [(See more)](https://github.com/swizzin/swizzin/blob/master/CONTRIBUTING.md#editor-plugins-and-tooling)
- [x] Prints to terminal are handled [(See more)](https://github.com/swizzin/swizzin/blob/master/CONTRIBUTING.md#printing-into-the-terminal)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] Testing was done
   - [x] Tests created or no new tests necessary
   - [ ] Tests executed

### ✅❎ Passed
amd64 on buster:

- Fresh install
- Update install (apt to tarball) -- data is moved to .config/Sonarr to allow purging sonarr to remove it from `dpkg -l`
- Removal

amd64 on bullseye:
- Fresh install
- Remove

### 🛠🛠 TODO
- Test ARM

### ❌❌ Currently failing

- None

### CODE REVIEW

**Comment by bakerboy448** [comment]
just for posterity

> bullseye does not have a repo to install from, yet it shouldn't matter

Believe focal's repo should work fine as a fallback.

> Moves sonarr to utilizing the tar ball installation

Officially not supported / advanced user yadda yadda.  but given the deps and everything will be handled by swizzin. should be fine.

**Comment by flying-sausages** [comment]
@liaralabs what about applying #719 first?

File: scripts/update/sonarr.sh [hunk 1]
```diff
@@ -42,3 +31,65 @@ if [[ -f /install/.sonarrv3.lock ]]; then
...
+    rm -f /etc/apt/sources.list.d/sonarr.list*
+    systemctl daemon-reload
+
+    #Restart sonarr because apt-removal stops it
+    if [[ $isActive == "active" ]]; then
+        systemctl start sonarr
+    fi
+
+    #Update broken symlink to old service
+    if systemctl is-enabled sonarr > /dev/null 2>&1; then
```
Comment (flying-sausages) [inline scripts/update/sonarr.sh:91]: My service was running but disabled after the update

**Comment by liaralabs** [comment]
> Believe focal's repo should work fine as a fallback.

Not a huge fan of this workaround.

>  but given the deps and everything will be handled by swizzin

deps were pulled directly from the sonarr deb package, so I believe it should be okay. If required deps somehow change before the dotnet version finally ships then we can associate that into the updater. However, I chose to use the exact deps rather than install `mono-complete` as the `mono-complete` package is massive and it's clear that sonarr doesn't require the full base to function.

File: scripts/update/sonarr.sh (hunk 1 above)
Comment (liaralabs) [reply scripts/update/sonarr.sh]: Updated to check the variable before hand and apply the fix based on previously reported status.

**Review by flying-sausages — APPROVED** [review]

**Pull request merged**